docs/index.html

```diff
@@ -1247,50 +1246,115 @@ <h4>Workflow Instance</h4>
-                    "VerifiablePresentation",
-                    "TraceablePresentation"
-                  ],
-                  "workflow": {
-                    "definition": [
-                      "urn:uuid:n1552885-cc91-4bb3-91f1-5466a0be084e"
-                    ],
-                    // This refers to a Workflow Instance
-                    "instance": [
-                      "urn:uuid:f5fb6ce4-b0b1-41b8-89b0-331ni58b7ee0"
-                    ]
-                  },
-                  "holder":{
-                    "id":"did:web:sender.example",
-                    "type":"Organization",
-                    "location":{
-                       "type":"Place",
-                       "geo":{
-                          "type":"GeoCoordinates",
-                          "latitude":"68.7083",
-                          "longitude":"4.6377"
-                       },
-                       "address":{
-                          "type":"PostalAddress",
-                          "organizationName":"Ratke - Bergstrom",
-                          "streetAddress":"21851 Ima Heights",
-                          "addressLocality":"O'Connellborough",
-                          "addressRegion":"Missouri",
-                          "postalCode":"65587",
-                          "addressCountry":"Cyprus"
-                       }
-                    }
-                 }
-                }
-                </pre
-          >
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3id.org/traceability/v1"
+  ],
+  "id": "urn:uuid:3978344f-8596-4c3a-a978-8fcaba3903c5",
+  "type":
+  [
+    "VerifiablePresentation",
+    "TraceablePresentation"
+  ],
+  "workflow": {
+    "definition": [
+      "urn:uuid:n1552885-cc91-4bb3-91f1-5466a0be084e"
+    ],
+    // This refers to a Workflow Instance
+    "instance": [
+      "urn:uuid:f5fb6ce4-b0b1-41b8-89b0-331ni58b7ee0"
+    ]
+  },
+  "holder":{
+    "id":"did:web:sender.example",
+    "type":"Organization",
+    "location":{
+        "type":"Place",
+        "geo":{
+          "type":"GeoCoordinates",
+          "latitude":"68.7083",
+          "longitude":"4.6377"
+        },
+        "address":{
+          "type":"PostalAddress",
+          "organizationName":"Ratke - Bergstrom",
+          "streetAddress":"21851 Ima Heights",
+          "addressLocality":"O'Connellborough",
+          "addressRegion":"Missouri",
+          "postalCode":"65587",
+          "addressCountry":"Cyprus"
+        }
+    }
+  }
+}
+        </pre>
+        </section>
+        <section>
+          <h4>Workflow Replace</h4>
+          <p>
+A holder MAY use the <code>replace</code> property of a <code>TraceablePresentation</code> 
+to communciate to a verifier that a previous presentation should be replaced with a new presentation.
+          </p>
+          <p>
+Holders SHOULD ensure the previous presentation was accepted before presenting a <code>replace</code> presentation.
```

Comment:
```suggestion
to communciate to a verifier that previous credentials that were presented should be removed or replaced.
          </p>
          <p>
Holders SHOULD ensure the previous presentation was accepted before presenting an <code>instructions</code> presentation.
<p>Within `instructions` properties are included that are each an array of credential IDs
for which the property indicates the operation that should be performed on those credentials.
Defined in this specification are operations named `update`, and `delete`.
</p>
```

Reply:
^ lets put this proposal in a separate PR, I prefer to address changes to the proposal in a follow on PR, its fine to suggest an issue marker.